src/pe/header.rs

```diff
@@ -1071,6 +1225,69 @@ mod tests {
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
     ];
 
+    static NO_RICH_HEADER: [u8; 262] = [
```

Comment:
this should be `const` not `static`

Reply:
Solved in 0e7c3778a552e507fe17801c3a82b9852de53542.
Since this is trivial changes, I'd like to make this change request solved on my end, however if you have anything else please reopen.